### PR TITLE
:gear:  Project set up and ERD implementation :100: 

### LINKED ISSUE #2: [3pts] Implement Food list (with all CRUD implementation, except for 'update')
- Should display a list of food added by the logged-in user as in the wireframe (for teams with only 2 people - display also quantity of a given food).
- Should lead to a form that allows users to add new food.

### LINKED ISSUE #10: [1pt] Project Setup
- Set up project
- Config Linter

### PR DESCRIPTION
**Pull request by SimonGideon**
# Description

- Set up project
- Config Linter
- Add database using psql `recipe_app_development`

# Related Issues
None
# Checklist

- [x] Set up project
- [x] Config Linter
- [x] Add database using psql `recipe_app_development`
- [x] Add the following models `user`, `Food Recipe`, `Recipe`, `Recipe Food`, `Food`
- [x] Created association and relationship between tables.  
- [x] Add controllers. 
- [x] Create the food Add and index page
- [x] Add styling to the foods.
- [X] Added n+1 in Foods 
`ttp://127.0.0.1:3000/foods` to see the food inde page
- [x] Food list:
   - [x] Should display a list of food added by the logged-in user as in the wireframe (for teams with only 2 people - display also quantity of a given food).
   - [x] Should lead to a form that allows users to add new food.  

# Preview
image

After migration
image

# Additional Notes
```Ruby Version 3.0.2```

Thanks for you review :slightly_smiling_face:

### CODE REVIEW

File: app/models/recipe.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+class Recipe < ApplicationRecord
+  has_many :recipe_foods
+  has_many :foods, through: :recipe_foods
```
Comment (amentetteh) [inline app/models/recipe.rb:3]: Consider adding missing attributes such as Cooking & Preparation time, description and public.

**Review by amentetteh — CHANGES_REQUESTED** [review]
Hi @SimonGideon,

## Highlights
✔️ No linter errors
✔️ Followed best practices
✔️ Working app
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/models/recipe.rb (hunk 1 above)
Comment (SimonGideon) [reply app/models/recipe.rb:3]: @amentetteh notted the requested changes will be included in the future commits.

Comment (SimonGideon) [reply app/models/recipe.rb:3]: image
The requested changes already exist int the recipe table.
Please check it :point_right: [here](https://github.com/amentetteh/ror-recipe-app/blob/project-set-up/db/schema.rb#L37-L47)

**Review by amentetteh — APPROVED** [review]
Hi @SimonGideon,

## APPROVED 🟢

## Highlights
✔️ No linter errors
Great implementation of all the requirements in this milestone. 👏 

_You have done an amazing job 👍_

Your project is complete!  There is nothing else to say other than... it's time to merge it. :shipit: 

Congratulations! 🎉 

## CHEERS AND HAPPY CODING!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by SimonGideon** [comment]
@amentetteh  THanks for the review :slightly_frowning_face:

**Pull request merged**